apps/ledger-live-desktop/src/renderer/modals/StartStake/index.tsx

```diff
@@ -28,6 +35,20 @@ const ModalStartStake: FC<ModalStartStakeProps> = ({ account, parentAccount, sou
   }
 
   useEffect(() => {
+    const platformAppRoute = getRouteToPlatformApp(account, walletState, parentAccount);
+
+    if (platformAppRoute) {
+      // Redirect to the platform app preferentially
+      dispatch(closeModal("MODAL_START_STAKE"));
+      history.push({
+        pathname: platformAppRoute.pathname,
+        state: {
+          ...platformAppRoute.state,
+        },
```

Comment:
Can we just do this here ?
```suggestion
        state: platformAppRoute.state,
```